source/2015-11-10-transparenz-als-standard.html.markdown.erb

```diff
@@ -0,0 +1,85 @@
+
+Das Ziel dabei war es, zu erörtern, ob eine öffentliche Transparenz für Panter
+als Unternehmen überhaupt Sinn machen würde und wie wir uns diese zu Nutzen
+machen könnten.
+
+<%= image_tag('kalamata-transparenz.jpg') %>
+
+Nach dem Brainstorming an der griechischen Sonne und unzähligen Post-it-Notizen
+wurde gemeinsam der Entschluss gefasst, dass Panter in Zukunft relevante Daten
+des Unternehmens öffentlich publizieren wird.  (Siehe auch Blog-Post [Panter,
+die demokratische
+Firma](http://blog.panter.ch/2015/06/12/panter-die-demokratische-firma.html))
+
+
+## Die Gründe, Offenheit zu praktizieren
+
+1. Durch das Offenlegen aller relevanten Informationen wird Vertrauen sowohl zu
+   unseren Kunden als auch zu den Mitarbeitern aufgebaut und langfristig
+   erhalten.
+
+1. Indem sich das Unternehmen mit dem Aufbereiten und Analysieren der Daten
+   beschäftigt, können Probleme frühzeitig erkannt und Risiken reduziert
+   werden.
+
+1. Dieser Prozess schafft Klarheit und bildet ein Fundament für aussagekräftige
+   Informationen. Entscheide aufgrund von Missinformation oder mutmasslichen
+   Annahmen können somit minimiert werden.
+
+1. Die pro-aktive Offenlegung von Daten führt dazu, dass diese nicht auf
+   Umwegen ausfindig gemacht werden müssen und somit kann der Veröffentlichung
+   von falschen Informationen vorgebeugt werden.
+
+1. Grundsätzlich soll mit dieser Vorgehensweise die Kultur der Offenheit
+   gestärkt und weitergeführt werden.
+
+1. Als Dienstleistungsunternehmen wollen wir als Vorbild für unsere Industrie
+   und unsere Kunden wirken und Open Data als einen grundlegenden Wert
+   etablieren.
+
+
+## Die Herausforderungen
+
+1. Trotz der Offenlegung von Informationen soll die Privatsphäre aller
+   Mitarbeiter gewahrt werden. Die veröffentlichten Daten sollen nicht
+   persönlicher Natur sein, sondern lediglich das Unternehmen als Gesamtheit
+   repräsentieren.
+
+1. Ein grundsätzliches Problem von Daten und Statistiken ist, dass sie einen
+   gewissen Interpretationsspielraum zulassen. Durch möglichst klares
+   Beschreiben der veröffentlichten Daten soll dieser minimiert werden.
+
+
+1. Eine weitere Herausforderung ist die Auswahl der relevanten Daten. Mit dem
+   heutigen Überangebot an Daten ist es schwierig und ebenso notwendig, diese
+   auf sinnvolle Weise zu filtern und nur die tatsächlich relevanten und
+   interessanten zu veröffentlichen. Überschwemmt man sich selbst und andere
+   mit Information, erfüllt man zwar seinen grundsätzlichen Zweck, jedoch ist
+   die langfristige Nützlichkeit nicht wirklich gewährleistet.
+
+Ein Teil der Daten wird nun schon mittels unserem
+[Dashboard](https://dashboard.panter.ch) in Echtzeit publiziert.
+
+<%= image_tag('dashboard.jpg') %>
+
+Wir sind momentan damit beschäftigt, alle relevanten Daten aufzubereiten und
+diese nach und nach zu veröffentlichen. Das Dashboard wird demzufolge laufend
+erweitert und die Statistiken in zukünftigen Blog-Posts genauer behandelt
+werden.
```

Comment:
@koffeinfrei I meant adding a link in the article, maybe something like this here:

```
Der Code hinter diesem Blog sowie Artikel im Entwurfsstadium sind ebenfalls
[öffentlich auf GitHub zugänglich](https://github.com/panter/blog.panter.ch).
```